### DISCUSSION TITLE
Can DP-Gen automatically train the model?

### DISCUSSION THREAD

**Discussion by ghost** [deepmodeling/dpgen#1280] [General]
I am quite new here. If I understand correctly, I am curious about how DP-Gen do in the run step. In contrast to DeepMD, where we use data from AIMD and split it into training and validation sets before training the model, DP-Gen seems to work differently. With DP-Gen, it appears that we can directly use the generated data to train the model, or it can automatically train the model for you. However, you're unsure about how to determine the training and validation sets in this case.

**Comment by wanghan-iapcm** [comment]
All data generated by dpgen are supposed to be used to train the model, as all the data are supposed to be critical to the accuracy. 
In practice, you may generate an independent validation set targeting at your application, or randomly split the dpgen training set for validation.

**Reply by ghost** [reply]
Thank you for your reply. However, I have a problem when I finished the init step. Under directory 02.md, where the result of short AIMD should be produced, there was no output at all. So, I was wondering if there is something wrong in my case because I have already checked my param.json file (below)
{
    "stages":           [1,2,3,4],
    "super_cell":       [1, 1, 1],
    "elements":         ["H","B", "C", "N", "F"],
    "from_poscar":      true,
    "from_poscar_path": "./POSCAR.emim_bf4",
    "potcars":          ["./POTCAR/POTCAR.H", "./POTCAR/POTCAR.B", "./POTCAR/POTCAR.C", "./POTCAR/POTCAR.N", "./POTCAR/POTCAR.F"],
    "relax_incar":      "./INCAR.rlx",
    "md_incar" :        "./INCAR.md",
    "scale":            [1.00],
    "skip_relax":       false,
    "pert_numb":        20,
    "md_nstep" :        10,
    "pert_box":         0.05,
    "pert_atom":        0.05,
    "coll_ndata":       5000,
    "_comment":         "that's all"
}

Thank you for your help.
